From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chris Bobbe <cbobbe@zulip.com>
Date: Thu, 22 Dec 2022 14:27:17 -0500
Subject: [upstream?] Use an interface instead of object type for
 ProxyNativeModule

This fixes errors like:

  Cannot extend ProxyNativeModule [1] with
  NotificationPermissionsModule because ProxyNativeModule [1] is not
  inheritable. [incompatible-use]
---
 .../build/NativeModulesProxy.types.js.flow            | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git types/expo-modules-core/build/NativeModulesProxy.types.js.flow types/expo-modules-core/build/NativeModulesProxy.types.js.flow
index 1d5048750..dbad5aa64 100644
--- types/expo-modules-core/build/NativeModulesProxy.types.js.flow
+++ types/expo-modules-core/build/NativeModulesProxy.types.js.flow
@@ -1,9 +1,8 @@
 /* @flow
  * @generated by TsFlower
  */
-export type ProxyNativeModule = {
-  [propertyName: string]: any,
-  addListener: (eventName: string) => void,
-  removeListeners: (count: number) => void,
-  ...
-};
+export interface ProxyNativeModule {
+  [propertyName: string]: any;
+  addListener: (eventName: string) => void;
+  removeListeners: (count: number) => void;
+}
-- 
2.32.0

